### ISSUE TITLE
Demo's for this module are broken in chrome

### ISSUE THREAD

**Issue by kmanoraj** [gilbarbara/react-joyride#382]
>Expected behavior
Popover should show the entire content

>Actual behavior
Popover is not loading properly.

>Steps to reproduce the problem
https://2zpjporp4p.codesandbox.io/
Happening in Chrome alone

Chrome Version: 67.0.3396.62

**Comment by kmanoraj** [comment]
Checked from multiple machines and looks like it is working fine except in my laptop. Not sure what is the issue. Attaching the screenshot to understand the issue better.
<img width="1296" alt="screen shot 2018-06-11 at 12 07 02 pm" src="https://user-images.githubusercontent.com/13959126/41215966-37a09f8e-6d70-11e8-94ae-16db41646a58.png">

**Comment by stuartherring** [comment]
I've encountered the same problem - it's not consistent - it'll work on some machines and not on others.  In fact, between two different Macs with the same version of Chrome it works on one, but not on the other.

But I've discovered that it appears to be related to the `drop-shadow` filter.  If you disable the filter on the box via the dev-tools, the problem goes away.  I tested that both with your example, and in my case too.  I'm assuming it must be a Chrome bug.
